### PR TITLE
Update all links to reflect style guide changes.

### PR DESCRIPTION
**Pull request by jeffayle**
Every link starts with http , ./ , ../ or `#`: 

```
$ grep -RPo '\]\((?!http|..?/|#).*?\)' src/
src/contributing/void-docs/style-guide.md:](example.md#heading-text)
```

No internal link (viz links that start with a dot) contains `html`:

```
$ grep -RPo '\]\((?=html)\..*\)' src/
[no output]
```

### CODE REVIEW

File: src/xbps/repositories/custom.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ For example, to define a remote repository:
 # echo 'repository=http://my.domain.com/repo' > /etc/xbps.d/my-remote-repo.conf
 ```
 
-> Note: Remote repositories need to be [signed](signing.md).
+> Note: Remote repositories need to be [signed](./signing.md).
 > [xbps-install(1)](https://man.voidlinux.org/xbps-install.1) refuses to install
 > packages from remote repositories if they are not signed.
```
Comment (ericonr) [inline src/xbps/repositories/custom.md:27]: This is probably something we should look at for #297 too!

**Comment by ericonr** [comment]
@flexibeast LGTM.

File: src/xbps/repositories/custom.md (hunk 1 above)
Comment (jeffayle) [reply src/xbps/repositories/custom.md:27]: I have the blockquote removed in #297, but that's it. was there something else you wanted done to it?

Comment (ericonr) [reply src/xbps/repositories/custom.md:27]: Just wanted to be sure :p You will probably need to rebase one of these, then.

File: src/contributing/void-docs/index.md [hunk 2]
```diff
@@ -3,5 +3,5 @@
 The sources for this handbook are hosted in the
 [void-docs](https://github.com/void-linux/void-docs) repository on
 [GitHub](https://github.com). If you would like to make a contribution Please
```
Comment (flexibeast) [inline src/contributing/void-docs/index.md:5]: ", please"

File: src/contributing/void-docs/index.md [hunk 3]
```diff
@@ -3,5 +3,5 @@
...
-follow our [style guide](./style-guide.md) and [submit](submitting.md) a
+follow our [style guide](./style-guide.md) and [submit](./submitting.md) a
 pull-request.
```
Comment (flexibeast) [inline src/contributing/void-docs/index.md:7]: "pull request"

**Comment by flexibeast** [comment]
Thank you @jeffayle and @ericonr! Merging.

File: src/contributing/void-docs/index.md (hunk 2 above)
Comment (flexibeast) [reply src/contributing/void-docs/index.md:5]: (i'll submit a separate PR for this.)

File: src/contributing/void-docs/index.md (hunk 3 above)
Comment (flexibeast) [reply src/contributing/void-docs/index.md:7]: (i'll submit a separate PR for this.)

**Pull request merged**